static/sass/styles.scss

```diff
@@ -1,4 +1,5 @@
 $color-tabs-active-bar: #e95420;
+$color-accent: #e95420;
```

Comment:
I think this is already defined in Vanilla, so maybe it is superfluous to re-define it here?

Reply:
I agree, this seems unnecessary.
```suggestion
```